docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -0,0 +1,1946 @@
+.. Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Zero Downtime Smart Contract Upgrades
+#####################################
+
+.. .. toctree::
+   :hidden:
+
+Theory of Operation
+===================
+
+What are Zero Downtime Smart Contract Upgrades?
+-----------------------------------------------
+
+Zero-Downtime Smart Contract Upgrades (SCU) are a new feature for Daml
```

Comment:
@hrischuk-da : I've just noticed that the SCU acronym is not yet present in our [Internal Glossary](https://docs.google.com/document/d/1npV7WUZh78I2SLLm-lbVvCsq7HWt5a-uqzZ7LeHhNJs/edit#bookmark=id.13s0zijoiyzs), can you add it please?

Reply:
Good point.  I will sync up and make it happen

Reply:
Done